tests/list.cpp

```diff
@@ -0,0 +1,109 @@
+#include <sys/stat.h>
+#include <iostream>
+#include <regex>
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+#include "bpftrace.h"
+#include "mocks.h"
+#include "list.h"
+
+namespace bpftrace {
+namespace test {
+namespace list {
+
+using ::testing::ContainerEq;
+using ::testing::StrictMock;
+using namespace std;
```

Comment:
No need for this if you're writing `std::` everywhere anyway. We use the explicit way (`std::vector` etc.) in the rest of the codebase, so prefer that.